examples/seismic/tutorials/13_making_synthetics.ipynb

```diff
@@ -0,0 +1,1504 @@
+    "\n",
+    "With our geological model defined, we will run a forward model for a single shot and generate a synthetic shot gather, before visualising the wavefield with PyVista.\n",
+    "\n",
+    "\n",
+    "## Creating our geological model:\n",
+    "\n",
+    "To begin, alongside GemPy, we need to import auxiliary modules:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import auxiliary modules\n",
+    "import numpy as np\n",
+    "\n",
+    "# %matplotlib qt5\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will now set up a GemPy `Model` object. This encapsulates the grid onto which the scalar fields associated with various surfaces are interpolated. Note that the extent is slightly greater than it will be for our Devito model (an extra half a grid spacing is added to each side). This is due to differences in the way in which grids are defined in each package and is necessary to ensure that the model is not stretched and distorted when transistioning between the two."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Active grids: ['regular']\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Set overarching model parameters\n",
+    "extent = (-5., 1005., -5., 1005., -1005., 5.)\n",
+    "shape = (101, 101, 101)\n",
+    "\n",
+    "geo_model = gp.create_model('Gempy-tutorial')\n",
+    "geo_model = gp.init_data(geo_model, extent=extent, resolution=shape)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Setting up Theano for our model (used by GemPy for interpolation)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Setting kriging parameters to their default values.\n",
+      "Compiling theano function...\n",
+      "Level of Optimization:  fast_compile\n",
+      "Device:  cpu\n",
+      "Precision:  float64\n",
+      "Number of faults:  0\n",
+      "Compilation Done!\n",
+      "Kriging values: \n",
+      "                        values\n",
+      "range             1749.371316\n",
+      "$C_o$            72864.285714\n",
+      "drift equations           [3]\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<gempy.core.interpolator.InterpolatorModel at 0x7f6620496110>"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "gp.set_interpolator(geo_model, output=['geology'], theano_optimizer='fast_compile')"
```

Comment:
I 'think' it was this line, but maybe add a comment noting that it may take a little while to execute?